Application/Api/Billing/Bookkeeping/ApiBasketRepayment.php

```diff
@@ -0,0 +1,360 @@
+        } else {
+            $FormContentLeft[] = new DatePicker('Basket[BillTime]', '', 'Rechnungsdatum');
+        }
+
+        if(!isset($_POST['Basket']['Creditor'])
+            && $CreditorList
+            && count($CreditorList) == 1){
+            $_POST['Basket']['Creditor'] = $CreditorList[0]->getId();
+        }
+
+        // choose between Add and Edit
+        $SaveButton = new Primary('Speichern', self::getEndpoint(), new Save());
+
+        // set Date to now
+        $Now = new \DateTime();
+        $Month = (int)$Now->format('m');
+        if(!isset($_POST['Basket']['Year'])){
+            $_POST['Basket']['Year'] = $Now->format('Y');
+        }
+        if(!isset($_POST['Basket']['Month'])){
+            $_POST['Basket']['Month'] = $Month;
+        }
+
+        $FormContentLeft[] = (new SelectBox('Basket[Year]', 'Jahr', $YearList))->setRequired();
+        $FormContentLeft[] = (new SelectBox('Basket[Month]', 'Monat', $MonthList, null, true, null))->setRequired();
+
+        $SaveButton->ajaxPipelineOnClick(self::pipelineSaveAddBasket($Identifier, $Type));
+        $Content = (new Form(new FormGroup(new FormRow(array(
+            new FormColumn(
+                new Panel($Type, $FormContentLeft, Panel::PANEL_TYPE_INFO)
+                , 6),
+            new FormColumn(
+                $this->getRightFormColumn()
+                , 6),
+            new FormColumn(
+                $SaveButton
+            )
+        )))))->disableSubmitAction();
+
+        /* @var Form $Content */
+        return $Content;
+    }
+
+    /**
+     * @return array
+     */
+    private function getRightFormColumn()
+    {
+
+        $Account = '(Standard) ';
+        $ToAccount = '(Standard) ';
+        if(($tblSettingAccount = Setting::useService()->getSettingByIdentifier(TblSetting::IDENT_FIBU_ACCOUNT))){
+            $Account .= $tblSettingAccount->getValue();
+        }
+        if(($tblSettingToAccount = Setting::useService()->getSettingByIdentifier(TblSetting::IDENT_FIBU_TO_ACCOUNT))){
+            $ToAccount .= $tblSettingToAccount->getValue();
+        }
+        $tblItemList = Item::useService()->getItemAll();
+        $SelectBox = new SelectBox('Basket[Item]', '', array('{{ Name }}' => $tblItemList));
+
+        return array(
+            new Panel('Beitragsart', array($SelectBox), Panel::PANEL_TYPE_INFO),
+            new Panel('Fibu'.new DangerText('*'), array(
+                new TextField('Basket[FibuAccount]', $Account, 'Konto'),
+                new TextField('Basket[FibuToAccount]', $ToAccount, 'Gegenkonto'),
+            ), Panel::PANEL_TYPE_INFO));
+
+    }
+
+    /**
+     * @param string $Identifier
+     * @param string $Type
+     * @param string $BasketId
+     * @param array  $Basket
+     *
+     * @return bool|Well
+     */
+    private function checkInputBasket(
+        $Identifier = '',
+        $Type = '',
+        $BasketId = '',
+        $Basket = array()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Basket" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)